SimPEG/EM/Static/DC/IODC.py

```diff
@@ -225,6 +222,12 @@ class IO(properties.HasProperties):
     n_spacing = None
     n_data = None
 
+    def __init__(self, **kwargs):
+        super(IO, self).__init__(**kwargs)
+        warnings.warn(
```

Comment:
@rowanc1, @sgkang: I moved the IO warning inside the instantiation of DC IO rather than on import (it was throwing a warning on the import of SimPEG, which was a bit annoying...)